components/templates/LoginTemplate.tsx

```diff
@@ -21,42 +22,48 @@ const LoginTemplate: React.FC = (): JSX.Element => {
   }
 
   return (
-    <SafeAreaView
-      style={[styles.container,
-        { backgroundColor: colors.backGround }]}>
+    <LinearGradient
+      colors={[colors.darkBlue, colors.backGround]}
+      start={[1, 1]}
+      end={[0, 0]}
+      flex={1}
+    >
+      <SafeAreaView
+        style={styles.container}>
 
-      <View gap={100} marginTop={30}>
-        <ContentTitle
-          title={t('screens.login.welcomeBack')}
-          subtitle={t('screens.login.loginPrompt')}
-        />
+        <View gap={120} marginTop={50}>
+          <ContentTitle
+            title={t('screens.login.welcomeBack')}
+            subtitle={t('screens.login.loginPrompt')}
+          />
 
-        <InputForm
-          visibleInputWithIcons={false}
-          visibleForgotPassword={true}
-        />
+          <InputForm
+            visibleInputWithIcons={false}
+            visibleForgotPassword={true}
+          />
 
-        <AuthButtonsGroup
-          onLoginPress={ButtonLogin}
-          onLoginGooglePress={ButtonLogin}
-        />
-      </View>
+          <AuthButtonsGroup
```

Comment:
Replace `AuthButtonsGroup` with [Separator](https://tamagui.dev/ui/separator)

```suggestion
          <Separator />
```